subgraphs.py

```diff
@@ -260,7 +258,7 @@ def make_diamond_subgraphs(self, sub_type, adj_list_in, adj_list_out):
                 for adj_entity in adj_list[entity]: # change depending on type
                     ent2 = adj_entity[0]
                     rel2 = adj_entity[1]
-                    if ent1 == ent2 or rel1 == rel2:
+                    if ent1 == ent2 or rel1 >= rel2:
```

Comment:
Check if changing the condition to `ent1 > ent2` or `ent1 < ent2` would further decrease the number of diamond subgraphs